<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RuleWidget</class>
 <widget class="QWidget" name="RuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>画线类型</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbLineType"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>最小人数</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbPersonMin"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>规则名称</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>报警轮巡(秒)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbAlarmLoop">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>3600</number>
       </property>
       <property name="singleStep">
        <number>60</number>
       </property>
       <property name="value">
        <number>60</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>最大人数</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbPersonMax"/>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>报警延时(秒)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbDelay">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>600</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>报警时间</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>开始时间</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTimeEdit" name="teStart">
            <property name="displayFormat">
             <string>hh:mm:ss</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>结束时间</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTimeEdit" name="teEnd">
            <property name="displayFormat">
             <string>hh:mm:ss</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>设定时间允许犯规次数</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sbAllowCount"/>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pbAdd">
          <property name="text">
           <string>添加</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbDel">
          <property name="text">
           <string>删除</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QListWidget" name="lwTimes"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>报警类型</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbAlarmType">
       <property name="insertPolicy">
        <enum>QComboBox::InsertAtBottom</enum>
       </property>
       <property name="minimumContentsLength">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>报警阀值(%)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbPercent">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>80</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
